README.md

```diff
@@ -197,6 +197,18 @@ The dependency options are listed below:
 | `.[tools]`       | Install dependencies for dedicated tools, such as quality classifiers.                       |
 | `.[sandbox]`     | Install all dependencies for sandbox.                                                        |
 
+- Install dependencies for specific OPs
+
+With the growth of the number of OPs, the dependencies of all OPs becomes very heavy. Instead of using the command `pip install -v -e .[sci]` to install all dependencies,
+we provide two alternative, lighter options:
+
+  - Automatic Minimal Dependency Installation: During the execution of Data-Juicer, minimal dependencies will be automatically installed. This allows for immediate execution, but may potentially lead to dependency conflicts.
+
+  - Manual Minimal Dependency Installation: To manually install minimal dependencies tailored to a specific execution configuration, run the following command:
+    ```shell
+    python tools/install_by_recipe.py --config path_to_your_data-juicer_config_file
```

Comment:
To keep consistent to Listing 1 of our DJ2.0 paper, rename it into `dj-install`

Reply:
For reference: https://github.com/modelscope/data-juicer/blob/a2612cdff70fcc282f31a62116d8837ebf551a3f/setup.py#L68-L72